### PR TITLE
Add disclaimer about using IPNS for websites

### PR DESCRIPTION
**Pull request by victorb**

### CODE REVIEW

File: examples/websites/README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ $ ipfs name publish $SITE_HASH
 Published to <your peer id>: /ipfs/$SITE_HASH
 ```
 
+(Disclaimer: IPNS is not production ready and assets could be loaded from two different resolved hashes when updating your website, leading to outdated/missing assets. Better to use IPFS hash directly to be sure the right asset is being loaded)
+
```
Comment (ghost) [inline examples/websites/README.md:49]: This won't be different when IPFS and IPNS are production ready. It's eventually consistent by its distributed nature, and you'll have to try to not involve the concept of time. One way to deal with this is similar to database migrations: you introduce the new thing, then point everything over, then remove the old thing, as opposed to doing it all in one atomic change. Front-end apps are a fun extension to this actually, you'll have to assume that all existing versions of your app run at the same time in some browser tab somewhere.

Btw, pointing to an IPFS hash directly doesn't help either, as there's stll DNS resolution with its TTLs and hidden caches involved :)

The thing that _indeed is_ annoying is IPNS's general performance for publish, lookup, and update propagation.

Comment (Kubuxu) [reply examples/websites/README.md:49]: Recent change in dht should improve DHT performance but it will be seen in practice.

Comment (VictorBjelkholm) [inline examples/websites/README.md]: Thanks @lgierth and @Kubuxu for the feedback. I've updated the disclaimer to mention this instead, as I think it's important to let people know about this.

**Comment by victorb** [comment]
Thanks @lgierth and @Kubuxu for the feedback. I've updated the disclaimer to mention this instead, as I think it's important to let people know about this.

**Comment by victorb** [comment]
@lgierth @Kubuxu good to merge?

**Pull request merged**